src/main.rs

```diff
@@ -440,3 +453,19 @@ fn print_avgs(options: &Options, avgs: &BTreeMap<String, avg::AVG>) {
         }
     }
 }
+
+/// Prints "Package {pkg} is affected by {issues}. {severity}!" colored
+fn print_avg_colored(t: &mut Box<term::StdoutTerminal>, pkg: &String, avg: &avg::AVG) {
+    t.reset().expect("term::stdout failed");
```

Comment:
This shouldn't be needed. If it's needed, then you forgot to reset the terminal somewhere.